extensions/ql-vscode/src/vscode-tests/no-workspace/query.test.ts

```diff
@@ -61,21 +59,27 @@ type QueryTestCase = {
 // Test cases: queries to run and their expected results.
 const queryTestCases: QueryTestCase[] = [
   {
-    queryPath: path.join(__dirname, '../data/query.ql'),
+    queryPath: path.join(baseDir, 'query.ql'),
     expectedResultSets: {
       '#select': [[42, 3.14159, 'hello world', true]]
     }
   },
   {
-    queryPath: path.join(__dirname, '../data/multiple-result-sets.ql'),
+    queryPath: path.join(baseDir, 'compute-default-strings.ql'),
```

Comment:
Looks great! The test failures are now from `files.test.ts`, which expected to find only 2 queries in this folder and now found the third one.

Reply:
Fixed now. Also, the sort was wrong also.